backend/api/Areas/Project/Mapping/Search/ProjectMap.cs

```diff
@@ -39,6 +40,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.CreatedOn, src => src.CreatedOn)
                 .Map(dest => dest.CreatedBy,
                     src => src.CreatedById != null ? src.CreatedBy.DisplayName : null)
+                .BeforeMapping((src, dest) => JsonConvert.PopulateObject(src.Metadata ?? "{}", src))
```

Comment:
The only concern I have here for the search is that we're returning an array of projects.  Ideally we won't do any extra serialization here if we don't need to.

Reply:
I'll test tasking this out and see if it breaks anything.